.github/workflows/check-milestone.yml

```diff
@@ -0,0 +1,28 @@
+name: Check Milestone
+# This check makes sure that the milestone is properly set.
+# To bypass this check, label the PR with "Long Term".
+on:
+  pull_request:
+    types: [opened, edited, labeled, unlabeled, synchronize]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'Long Term') == 0 }}
+    steps:
+      - name: Get PR number and milestone
+        run: |
+          echo "PR_NUMBER=${{ github.event.pull_request.number }}" >> $GITHUB_ENV
+          echo "TAGGED_MILESTONE=${{ github.event.pull_request.milestone.title }}" >> $GITHUB_ENV
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+      - name: Check the milestone
+        run: |
+          set -xe
+          if [ $TAGGED_MILESTONE == "" ]
```

Comment:
Since we use `TAGGED_MILESTONE` can check it here:

```suggestion
          if [ "${{ github.event.pull_request.milestone.title }}" == "" ]
```

and remove the "Get PR number and milestone" task?